packages/microcosm-www-next/src/components/graphic.js

```diff
@@ -1,33 +1,50 @@
 import React from 'react'
 import BrowserGraphic from './browser-graphic'
 
-const Graphic = ({ section, microcosmView }) => (
-  <figure
-    id={'graphic-' + section}
-    className="section__graphic__figure"
-    data-module="ObserveGraphic"
-    data-section={section}
-  >
-    <div
-      className={
-        'flip-container' +
-        (microcosmView || section === 0 || section == 9 ? '' : ' -flipped')
-      }
-    >
-      <div className="flipper">
-        <div className="flipper__front">
-          <img
-            data-src={`/${section}-microcosm.png`}
-            className="lazyload microcosm-graphic"
-            alt="TODO"
-          />
-        </div>
-        <div className="flipper__back">
-          <BrowserGraphic />
-        </div>
-      </div>
-    </div>
-  </figure>
-)
+export default class Graphic extends React.Component {
+  shouldComponentUpdate(nextProps) {
+    let fadeClassChanged = nextProps.fadeClass !== this.props.fadeClass
+    let viewChanged = nextProps.microcosmView !== this.props.microcosmView
```

Comment:
are you part of the anti `const` camp?

Reply:
lol no strong feelings either way, which team should I be on?? I can change it to `const`

Reply:
it's a war as fierce as tabs and spaces. You do you haha